packages/console/src/panel.ts

```diff
@@ -55,7 +55,9 @@ export class ConsolePanel extends MainAreaWidget<Panel> {
       options.contentFactory || ConsolePanel.defaultContentFactory);
     const count = Private.count++;
     if (!path) {
-      path = `${basePath || ''}/console-${count}-${UUID.uuid4()}`;
+      path = `${
+        basePath ? basePath + '/' : ''
```

Comment:
I'm not sure if `basePath` can have a trailing `/`, and if we should check for it.

Reply:
Does the git history say anything about why the `/` was introduced in the first place?

Reply:
Looks like it was introduced in https://github.com/jupyterlab/jupyterlab/commit/50a771bc9affed9e2dbaf5bbe459f4f7b52ddad6. [The PR](https://github.com/jupyterlab/jupyterlab/pull/1927) doesn't say much about it.

Reply:
Actually, `basePath` is the working directory where the console is launched, and I checked that it doesn't have a trailing slash, so this should be good.

Reply:
Maybe the join should be done with `URLExt.join` from `@jupyterlab/coreutils` instead? I think it can handle trailing slashes.

Reply:
Indeed, thanks!